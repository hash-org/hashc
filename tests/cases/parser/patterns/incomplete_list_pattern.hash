// run=fail, stage=parse

[a := b;
