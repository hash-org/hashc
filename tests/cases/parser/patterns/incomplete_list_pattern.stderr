error: encountered unclosed delimiter `[`, add a `]` after the inner expression
 --> $DIR/incomplete_list_pattern.hash:3:1
2 |   
3 |   [a := b;
  |   ^ here
